src/main/java/org/folio/services/MailService.java

```diff
@@ -0,0 +1,10 @@
+package org.folio.services;
+
+import io.vertx.core.Future;
+import io.vertx.core.json.JsonObject;
+import org.folio.rest.jaxrs.model.Configurations;
+import org.folio.rest.jaxrs.model.EmailEntity;
+
+public interface MailService {
```

Comment:
plz, use vertx event bus for async services

Reply:
all interface methods must have javadocs

Reply:
thanks.
made changes to the code.